vortex-file/src/writer.rs

```diff
@@ -69,27 +69,28 @@ impl VortexWriteOptions {
         )?;
 
         // First we write the magic number
-        let mut write = futures::io::Cursor::new(write);
-        write.write_all(MAGIC_BYTES).await?;
+        let mut writer = futures::io::Cursor::new(writer);
```

Comment:
Typically keep read / write because that aligns with the traits Read and Write

Reply:
I don't feel strongly about it, its just a `write` variable inside of a `write` function, will revert.